tutorials/scripting/c_sharp/c_sharp_variant.rst

```diff
@@ -9,17 +9,68 @@ For a detailed explanation of Variant in general, see the :ref:`Variant <class_V
 We recommend avoiding ``Godot.Variant`` unless it is necessary to interact with untyped engine APIs.
 Take advantage of C#'s type safety when possible.
 
-Any of ``Variant.As{TYPE}`` methods or the generic ``Variant.As<T>`` method can be used to convert
-a ``Godot.Variant`` to a C# type. Since the ``Godot.Variant`` type contains implicit conversions
-defined for all the supported types, calling these methods directly is usually not necessary.
+Converting from a Variant-compatible C# type to ``Godot.Variant`` can be done using implicit
+conversions. Also available are ``CreateFrom`` method overloads and the generic ``Variant.From<T>``
```

Comment:
```suggestion
conversions. You also have ``CreateFrom`` method overloads and the generic ``Variant.From<T>``
```
Or
```suggestion
conversions. There are also ``CreateFrom`` method overloads and the generic ``Variant.From<T>``
```
"Also available are" sounds clunky in English IMO

Reply:
FWIW to me (en-us) it's a common phrase that implies the first option is the default. `also available` shows up elsewhere in the docs too, just not followed with `are` in particular. I do think `There are also` works fine though, happy to change it.